### PR TITLE
Enable automerge for Jenkins controller and agent upgrade

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: renovate/flux.json [hunk 1]
```diff
@@ -21,13 +21,26 @@
...
+      "automerge": true,
+      "automergeType": "pr",
+      "platformAutomerge": true
+    },
+    {
+      "matchPackageNames": ["hmctspublic.azurecr.io/jenkins/jenkins"],
+      "automerge": true,
+      "automergeType": "pr",
+      "platformAutomerge": true,
+      "automergeSchedule": ["before 8am every weekday"]
```
Comment (timja) [inline renovate/flux.json:36]: update out of hours only

File: renovate/flux.json [hunk 2]
```diff
@@ -21,13 +21,26 @@
         "hmctsprivate.azurecr.io/camunda/optimize"
       ],
       "enabled": false
+    },
+    {
+      "matchPackageNames": ["hmcts/jenkins-packer"],
```
Comment (timja) [inline renovate/flux.json:26]: safe to update whenever as it's just updating the agent version for new ones spawning

File: renovate/flux.json [hunk 3]
```diff
@@ -21,13 +21,26 @@
...
+      "platformAutomerge": true,
+      "automergeSchedule": ["before 8am every weekday"]
     }
   ],
   "regexManagers": [
     {
       "description": "Update Jenkins image",
       "fileMatch": [
-        "jenkins\\.yaml$"
+        "jenkins\\.yaml$", "jenkins-controller-version\\.yaml$"
```
Comment (timja) [inline renovate/flux.json:43]: need to move to a separate file so CODEOWNERS can be disabled on it

File: renovate/flux.json [hunk 4]
```diff
@@ -39,7 +52,7 @@
     {
       "description": "Update Jenkins image gallery version",
       "fileMatch": [
-        "jenkins\\.yaml$"
+        "jenkins\\.yaml$", "jenkins-azure-vm-agent\\.yaml$"
```
Comment (timja) [inline renovate/flux.json:55]: need to move to a separate file so CODEOWNERS can be disabled on it

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**